src/SFA.DAS.AANHub.Application/Employers/Queries/GetEmployerMemberResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Employers.Queries
+{
+    public class GetEmployerMemberResult
+    {
+        public Guid MemberId { get; set; }
+        public string Email { get; set; } = null!;
+        public string Name { get; set; } = null!;
+        public string Organisation { get; set; } = null!;
+
+        public static implicit operator GetEmployerMemberResult?(Employer? employer)
```

Comment:
I am not liking this null checks, if the argument is made not nullable then we can avoid this null checks.

Reply:
So, I think the reason we need to do this is that ValidationBehaviour doesn't accept null result types yet.
` return new ValidatedResponse<GetEmployerMemberResult?>(employer);`

So this wouldn't work. I agree with the nullability checks though, they need to be removed.